c++/src/kj/common.h

```diff
@@ -1603,8 +1603,7 @@ class Deferred {
   inline ~Deferred() noexcept(false) { if (!canceled) func(); }
   KJ_DISALLOW_COPY(Deferred);
 
-  // This move constructor is usually optimized away by the compiler.
```

Comment:
The comment is still accurate (it's referring to the constructor, not the use of `kj::mv`).

Reply:
gotcha, fixing